packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -1268,6 +1273,30 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
 		);
 	});
 
+	it("get pending blob without close while online", async function () {
```

Comment:
What's being tested here? What you're doing is something we don't want people to do (using stash blob when the original container hasn't been closed yet)

Reply:
Got it, I'll remove this test then.